cmd/goreleaser/internal/configure.go

```diff
@@ -45,6 +46,7 @@ var (
 	winContainerArchs = []string{"amd64"}
 	darwinArchs       = []string{"amd64", "arm64"}
 	k8sArchs          = []string{"amd64", "arm64", "ppc64le", "s390x"}
+	ebpfProfilerArchs = []string{"amd64", "arm64"}
```

Comment:
I'm probably missing something; why are we building the eBPF distro for ARM64 here if it doesn't work according to the [comment above](https://github.com/open-telemetry/opentelemetry-collector-releases/pull/908/files#r2038983667)?

Reply:
Good point. It's added to goreleaser, but not used by the action afterward. I've removed it.